Backend/lib/shared-services/package.yaml

```diff
@@ -1,3 +1,8 @@
+# hpack defaults
```

Comment:
skip this diff?

Reply:
This is generated by pre-commit hook, it will keep on coming every time.